R/s3utils.R

```diff
@@ -144,3 +144,7 @@ s3_file_exists <- function(s3_path) {
   )
   return(exists)
 }
+
+dot_file_ext <- function(path) {
```

Comment:
it feels like there should be a better way of doing this, but it does seem like this is a necessary consequence of `tempfile()` not intelligently adding a dot to the file extension if needed so should be fine